las/client.py

```diff
@@ -128,20 +128,26 @@ def put_document(document_path: str, content_type: str, presigned_url: str) -> s
         :type content_type: str
         :param presigned_url: Presigned upload url from :py:func:`~las.Client.post_documents`
         :type presigned_url: str
+        :param use_kms: Adds KMS header to the request to S3. Set to true if your API using KMS default encryption on
+        the data bucket
```

Comment:
Is this correct english? 

I would prefer: 
Set to true if your API **is** using KMS default encryption on the data bucket

Reply:
Corrected.